### DISCUSSION TITLE
Documentation/Demo regarding syncing rotation and position with Client Side Prediction V2

### DISCUSSION THREAD

**Discussion by Toernblom** [FirstGearGames/FishNet#603] [Documentation]
Hey, I've had some issues following the documentation for CSP2. Getting the basic movement to work wasn't an issue at all, however. When i tried to do the same for rotation it wasnt as easy as i thought it was going to be. I found it to be quite ambigious aswell as if replication and reconciliation were to run on all connected clients. I know it is now. But a deeper guide into a just syncing position and rotation and when to use each of the states would help a ton!

**Comment by maxkratt** [comment]
I absolutely agree with this, one of the biggest hurdles currently when moving forward from the documentation to making your own controller is handling rotation. It doesn't have to be a full FPS setup demo, even a top-down view with rotation integral to the movement would be hugely appreciated!

**Comment by Toernblom** [comment]
Another thing that would really need some explaining is CSP2 behaves differently ( might be me but not clearly documented ) depending on if you only have one editor up that acts both as a client and server or if you have two editors whereas one is client and one is a server.

**Comment by FirstGearGames** [comment]
The P2 demo is going to have a lot of new stuff.
We(99% anewbie) also have a working Kinematic controller in FishMMO.

Let's not get too crazy on some requests but Id love to hear what some simple but useful things would be.
